owtf/db/database.py

```diff
@@ -6,13 +6,18 @@
 """
 import functools
 import logging
+import sys
 
-from sqlalchemy import create_engine, exc, func
+from sqlalchemy import create_engine, func
 from sqlalchemy.orm import Session as _Session
 from sqlalchemy.orm import sessionmaker
 
 from owtf.db.models import Base
-from owtf.settings import DATABASE_IP, DATABASE_NAME, DATABASE_PASS, DATABASE_PORT, DATABASE_USER
+from owtf.settings import DATABASE_IP, DATABASE_NAME, DATABASE_PASS, DATABASE_USER
+from owtf.utils.logger import OWTFLogger
+
+logger = OWTFLogger()
```

Comment:
No need to import the logger. Just import the existing instance from `core.py`.

Reply:
what do you exactly mean by import existing instance from core.py? because as of now core does not export `logger` also db connection is called even before executing core.

Reply:
Then maybe just `logger = logging.getLogger(__name__)` should do it.

Reply:
But we won't get command line logging formatting that way, is there a problem with the current approach?

Reply:
I am thinking about unnecessary code duplication here. Can we configure the custom logger in `utils/logger.py` which can be used universally without initializing it everywhere?

Reply:
Let me have a look at it, I will get back to you on this.

Reply:
How about if we do
```python
logger = OWTFLogger()
```
in logger.py and then instead of importing OWTFLogger we can import `logger` then we will only need to enable_logger in the importing model to get what we want.